stp_zmq/zstack.py

```diff
@@ -1163,10 +1166,16 @@ def retryDisconnected(self, exclude=None):
         for name, remote in self.remotes.items():
             if name in exclude or remote.isConnected:
                 continue
-            if remote.socket:
-                self.sendPingPong(remote, is_ping=True)
-            else:
+
```

Comment:
I am not seeing how this retry mechanism is tested by the above tests, this retry mechanism should be tested by patching one stack's ping pong handler to not send pong for any ping and see that retry attempts are counted and after `MAX_RECONNECT_RETRY_ON_SAME_SOCKET*RETRY_TIMEOUT_*` time, the other stack does a `reconnectRemote`, in another test patch should be reverted under `t` where `RETRY_TIMEOUT_*
 < t < MAX_RECONNECT_RETRY_ON_SAME_SOCKET*RETRY_TIMEOUT_`